wagtail/admin/templates/wagtailadmin/account/account.html

```diff
@@ -18,6 +18,15 @@
                 </small>
 
             </li>
+            <li class="row row-flush">
+                <div class="col6">
+                    <a href="{% url 'wagtailadmin_account_change_email' %}" class="button button-primary">{% trans "Change email" %}</a>
+                </div>
+
+                <small class="col6">
+                    {% trans "Change email address linked to your account." %}
+                </small>
+            </li>
             {% if show_change_password %}
```

Comment:
I see other blocks are wrapped in a `if show` conditional like this – would we need to do the same for email?

Reply:
Hey Thibaud, added the screenshot of the link in account settings and for the form. 
![image](https://user-images.githubusercontent.com/3624620/37457956-b70163de-2843-11e8-9602-2f5c854e5ef7.png)
![image](https://user-images.githubusercontent.com/3624620/37457995-d7d1ac18-2843-11e8-8c2b-e0fa2fbc53f8.png)

Tested it on Chrome/Safari/Firefox
On the `wagtail/admin/templates/wagtailadmin/account/account.html` you can see a few `if` statements which are used for: 
*Password: When a user doesn't have a usable password, or password was never set, password can't be changed (see `has_usable_password` )
*Notifications: When a user doesn't have editing rights, so notifications aren't needed since none of his posts will be edited

From what I know, and confirming it from the `User Edit form`, changing email doesn't have any restriction. 

Cheers!